site/docs/samples/notebook-read.md

```diff
@@ -61,7 +61,13 @@ Make a note of the service endpoint, bucket name, and access credentials. You wi
 
 In this step you are performing the role of the data owner, registering his data in the data catalog and registering the credentials for accessing the data in the credential manager.
 
-Register the credentials required for accessing the dataset as a kubernetes secret. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+In this tutorial, we assume that OpenMetadata is used as the data catalog. Datasets can be registered either directly, through the OpenMetadata UI, or indirectly, through the data-catalog connector.
+
+If you use OpenMetadata directly through the UI, and use its discovery mechanism to add asset tables, you can determine an asset ID through the corresponding OpenMetadata table URL. If, for instance, the URL for the table is: `http://localhost:8585/table/openmetadata-s3.default.demo."PS_20174392719_1491204439457_log.csv"`, then the corresponding asset ID is `"openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""`.
```

Comment:
Please provide screen captures and instructions on how to do it via the UI

Reply:
I added a link to a separate page with the screen captures.